### PR TITLE
Binaryedge ports

### PR DESCRIPTION
**Pull request by MrJester**

### CODE REVIEW

File: modules/recon/hosts-ports/binaryedge_ports.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    def module_run(self, ipaddrs):
+        key = self.get_key('binaryedge_api')
+        for ipaddr in ipaddrs:
+            self.heading(ipaddr, level=0)
+            resp = self.request(url=f"https://api.binaryedge.io/v2/query/ip/{ipaddr}", headers={'X-Key': key})
+            if resp.status_code == 200:
+                for event in resp.json().get('events'):
+                    for result in event['results']:
+                        self.insert_ports(ip_address=ipaddr, port=result['target']['port'],
+                                          protocol=result['target']['protocol'])
```
Comment (lanmaster53) [inline modules/recon/hosts-ports/binaryedge_ports.py]: Looks good. Please add a newline to the end of the file and update the pull request. Thanks!

Comment (MrJester) [reply modules/recon/hosts-ports/binaryedge_ports.py]: Should be updated with the newline now.

**Comment by lanmaster53** [comment]
Looks good, but need to update the index for this module too. Thanks!

**Comment by MrJester** [comment]
Updated this branch with the index

File: modules.yml [hunk 2]
```diff
@@ -1020,3 +1020,13 @@
   path: recon/netblocks-hosts/reverse_resolve
   required_keys: []
   version: '1.0'
+- author: Ryan Hays
+  dependencies: []
+  description: Uses the BinaryEdge API to discover open services for IP Addresses.
+  files: []
+  last_updated: '2019-05-28'
+  name: BinaryEdge.io Ports lookup
+  path: dev/binaryedge_ports
```
Comment (lanmaster53) [inline modules.yml]: Same here as the other.

File: modules.yml [hunk 3]
```diff
@@ -1020,3 +1020,13 @@
...
+  required_keys:
+  - binaryedge_api
+  version: '1.0'
```
Comment (lanmaster53) [inline modules.yml]: Same here as the other.

Comment (MrJester) [reply modules.yml]: try it now

**Pull request merged**